presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -93,6 +93,7 @@
 
     private boolean useOrcColumnNames;
     private boolean orcBloomFiltersEnabled;
+    private Double orcBloomFilterFpp = 0.05;
```

Comment:
Since we have a default, I think we can leave this as a primitive. That's what we do everywhere else.